src/main/kotlin/lib/graphics/turtle/awt/GraphicsTurtleCore.kt

```diff
@@ -15,14 +16,17 @@ import kotlin.math.sin
  * [TurtleCore] interface over AWT [Graphics2D]
  */
 class GraphicsTurtleCore(
-    graphics: Graphics2D,
+    graphics: Graphics,
     private val size: Dimension,
     private val stateView: TurtleStateView
 ) : TurtleCore {
     private val graphics = object {
+        // Every Graphics in runtime is actually Graphics2D
+        private val g2d = graphics as Graphics2D
+
         // Force all interactions with Graphics2D to happen in  UI thread
         operator fun invoke(block: Graphics2D.() -> Unit): Unit = invokeAndWait {
-            graphics.block()
+            g2d.block()
         }
     }
 
```

Comment:
```suggestion
    override fun line(x1: Double, y1: Double, x2: Double, y2: Double): Unit =
        graphics {
            color.let {
                color = Color.BLACK
                drawLine(x1.roundToInt(), y1.roundToInt(), x2.roundToInt(), y2.roundToInt())
                color = it
            }
        }
```
И будут тебе чёрные линии.

Reply:
Поправил. @gumanoid глянешь?